### ISSUE TITLE
Support GitHub Actions

### ISSUE THREAD

**Issue by mgedmin** [mgedmin/check-python-versions#22]
A bunch of ZopeFoundation repos are getting GitHub actions based on https://github.com/zopefoundation/meta/blob/master/config/default/tests.yml.in:

```yaml
jobs:
  build:
    strategy:
      matrix:
        config:
        # [Python version, tox env]
        - ["3.8",   "lint"]
        - ["2.7",   "py27"]
        - ["3.5",   "py35"]
        - ["3.6",   "py36"]
        - ["3.7",   "py37"]
        - ["3.8",   "py38"]
        - ["3.9",   "py39"]
        - ["pypy2",   "pypy"]
        - ["pypy3",   "pypy3"]
        - ["3.8",   "coverage"]
```

One project that does this is zope.browser.

_Originally posted by @mgedmin in https://github.com/mgedmin/check-python-versions/issues/9#issuecomment-731046810_

**Comment by mgedmin** [comment]
mypy has an interesting workflow:
https://github.com/python/mypy/blob/63f2fed5d0e10d4d875442e9bba2283c81dfdf4b/.github/workflows/test.yml#L22-L34
```yaml
jobs:
  build:
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        name: [windows-py37-32, windows-py37-64]
        include:
        - name: windows-py37-32
          python: '3.7'
          arch: x86
          os: windows-latest
          toxenv: py37
        - name: windows-py37-64
          python: '3.7'
          arch: x64
          os: windows-latest
          toxenv: py37
```

It is very verbose.  I don't like it.

**Comment by mgedmin** [comment]
pytest has the same sort of thing, very verbose:
https://github.com/pytest-dev/pytest/blob/91dbdb60930e1b573de366cbe7be6dac77ba200f/.github/workflows/main.yml#L20-L122

**Comment by mgedmin** [comment]
pyramid uses a matrix on `py`:
https://github.com/pytest-dev/pytest/blob/91dbdb60930e1b573de366cbe7be6dac77ba200f/.github/workflows/main.yml#L20-L122

**Comment by mgedmin** [comment]
Fixed in bce35261ef70990c4e9cc90a86a92650a05078f1.

check-python-versions 0.17.0 has support for GitHub Actions!
